integration-cli/docker_api_inspect_test.go

```diff
@@ -42,3 +43,33 @@ func (s *DockerSuite) TestInspectApiContainerResponse(c *check.C) {
 		c.Fatalf("Path of `true` should not be converted to boolean `true` via JSON marshalling")
 	}
 }
+
+func (s *DockerSuite) TestInspectApiImageResponse(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "pull", "busybox")
+	out, _, err := runCommandWithOutput(runCmd)
+
+	if err != nil {
+		c.Fatalf("failed to pull image: %s, %v", out, err)
+	}
+
+	runCmd = exec.Command(dockerBinary, "images", "-q", "busybox")
```

Comment:
All I meant is that they could use 'busybox' (by name) instead of busybox's ID.  Should result in the same thing, right?